app/scripts/modules/netflix/fastProperties/domain/canary.domain.ts

```diff
@@ -0,0 +1,87 @@
+
+import {PropertyCommand} from './propertyCommand.model';
+import {IUser} from 'core/authentication/authentication.service';
+import {PropertyCommandType} from './propertyCommandType.enum';
+import {ICanaryDeployment} from './canaryDeployment.interface';
+import {AcaTaskStageConfigDetails} from './acaTaskStageConfigDetails.model';
+
+export class Canary {
+  public application: string = 'spinnakerfp';
+  public owner: string;
+  public canaryConfig: CanaryConfig;
+  public watchers: string[] = [];
+  public canaryDeployments: CanaryDeployments[];
+
+  constructor(private user: IUser, private command: PropertyCommand, private configDetails: AcaTaskStageConfigDetails ) {
```

Comment:
Ah, ok.  So if you provide access modifiers on ctor args, the Typescript compiler will bind them as class-level properties with the same access modifier.  But, if you don't actually reference the properties, then the access modifier is not needed (and indeed, the new versions of the linter will flag this as an error).

I propose adopting adopting the recommendation of the linter in that only apply the access modifier as necessary.  Note that this is not to be confused with class method access modifiers.